spec/services/book_to_nomis_config_spec.rb

```diff
@@ -80,6 +80,14 @@
     end
   end
 
+  shared_context 'offender restrictions working' do
+    before do
+      allow(Nomis::Feature).
+        to receive(:offender_restrictions_enabled?).and_return(true)
```

Comment:
Yes, but that method controls the environment variables configuration which is lower level.

`Nomis::Feature` decides if all dependencies for a given feature are met, which I think it makes it nicer to stub in this case as this test doesn't need to care about all the dependencies that make a feature enabled.

For example for offender restrictions the offender lookup also needs to be enabled.